clang/unittests/Format/FormatTestTableGen.cpp

```diff
@@ -40,6 +40,13 @@ class FormatTestTableGen : public ::testing::Test {
     EXPECT_EQ(Code.str(), format(Code)) << "Expected code is not stable";
     EXPECT_EQ(Code.str(), format(test::messUp(Code)));
   }
+
+  static void verifyFormat(llvm::StringRef Code, const FormatStyle &Style) {
```

Comment:
It seems like this file was never brought up to date with all the other tests when f8d10d5ac9ab4b45b388c74357fc82fb96562e66 happened

Reply:
I have not noticed that. Thank you. I can also fix this. But now I am planing whether (and how) to split this growing pull request. A kind of refactoring may be better done after that.